utils/products.js

```diff
@@ -1,152 +1,664 @@
       "https://github.com/eclipse-tractusx/portal",
       "https://github.com/eclipse-tractusx/portal-backend",
       "https://github.com/eclipse-tractusx/portal-frontend",
       "https://github.com/eclipse-tractusx/portal-frontend-registration",
       "https://github.com/eclipse-tractusx/portal-assets",
-      "https://github.com/eclipse-tractusx/portal-iam"
+      "https://github.com/eclipse-tractusx/portal-iam",
+      "https://github.com/eclipse-tractusx/portal-shared-components",
     ],
     committers: [
       "https://github.com/evegufy",
       "https://github.com/ntruchsess",
       "https://github.com/oyo",
-      "https://github.com/Phil91"
+      "https://github.com/Phil91",
     ],
-    mailTo: "tractusx-dev@eclipse.org?subject=Request Portal and Marketplaces Team",
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request Portal and Marketplaces Team",
     hasBoard: false,
-    showVersion: true
+    showVersion: true,
+  },
+  {
+    productName: "Predictive Unit Real-Time Information Service",
+    productDescription:
+      "Predictive Unit Real-Time Information Service (PURIS) for Short Term Demand and Capacity Management",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/puris",
+      "https://github.com/eclipse-tractusx/puris-backend",
+      "https://github.com/eclipse-tractusx/puris-frontend",
+    ],
+    committers: [
+      "https://github.com/tom-rm-meyer-ISST",
+      "https://github.com/mhellmeier",
+      "https://github.com/tomaszbarwicki",
+    ],
+    mailTo: "tractusx-dev@eclipse.org?subject=Request puris Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "quality-dashboard",
+    productDescription: "TBD",
+    githubRepo: ["https://github.com/eclipse-tractusx/quality-dashboard"],
+    committers: ["https://github.com/tomaszbarwicki"],
+    mailTo: "tractusx-dev@eclipse.org?subject=Request quality-dashboard Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "Tractus-X infrastructure",
+    productDescription:
+      "Repository to manage the infrastructure of the Tractus-X project.",
+    githubRepo: ["https://github.com/eclipse-tractusx/sig-infra"],
+    committers: [
+      "https://github.com/evegufy",
+      "https://github.com/stephanbcbauer",
+    ],
+    mailTo: "tractusx-dev@eclipse.org?subject=Request sig-infra Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "Tractus-X release planning repository",
+    productDescription:
+      "Repository to manage the release planning of the Tractus-X project on a feature level.",
+    githubRepo: ["https://github.com/eclipse-tractusx/sig-release"],
+    committers: [
+      "https://github.com/tomaszbarwicki",
+      "https://github.com/stephanbcbauer",
+    ],
+    mailTo: "tractusx-dev@eclipse.org?subject=Request sig-release Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "Tractus-X security",
```

Comment:
Similar to the other "non-products", that @evegufy mentioned, I also wouldn't add the Security SIG.
If committer info for SIGs is valuable, consider adding them in the "Communities / SIGs" pages: https://eclipse-tractusx.github.io/community/sigs/intro